homeassistant/components/fully_kiosk/notify.py

```diff
@@ -0,0 +1,36 @@
+"""Support for fully_kiosk notifications."""
+
+from typing import Any
+
+from homeassistant.components.notify import BaseNotificationService
+from homeassistant.const import CONF_ENTITY_ID
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType, DiscoveryInfoType
+
+from .const import DOMAIN
+from .coordinator import FullyKioskDataUpdateCoordinator
+
+
+async def async_get_service(
```

Comment:
We're phasing out `async_get_service` in favor of implementing `NotifyEntity`

Reply:
Done. But how can I add it to a [notify group](https://www.home-assistant.io/integrations/group/#notify-groups) or in the `notifiers` of the [alert ](https://www.home-assistant.io/integrations/alert/) integration?

Reply:
This is a migration that is in progress. Eventually all things should only be entities.